pytket/binders/circuit/pool.cpp

```diff
@@ -0,0 +1,196 @@
+  pool_m.def(
+      "CSXdg_using_CX", &CircPool::CSXdg_using_CX,
+      "Equivalent to CSXdg, using CX and single-qubit gates");
+  pool_m.def(
+      "CSWAP_using_CX", &CircPool::CSWAP_using_CX,
+      "Equivalent to CSWAP, using CX and single-qubit gates ");
+  pool_m.def(
+      "ECR_using_CX", &CircPool::ECR_using_CX,
+      "Equivalent to ECR, using CX, Rx and U3 gates ");
+  pool_m.def(
+      "ZZMax_using_CX", &CircPool::ZZMax_using_CX,
+      "Equivalent to ZZMax, using CX, Rz and U3 gates ");
+  pool_m.def(
+      "CRz_using_CX", &CircPool::CRz_using_CX,
+      "Equivalent to CRz, using CX and Rz gates");
+  pool_m.def(
+      "CRx_using_CX", &CircPool::CRx_using_CX,
+      "Equivalent to CRx, using CX, H and Rx gates");
+  pool_m.def(
+      "CRy_using_CX", &CircPool::CRy_using_CX,
+      "Equivalent to CRy, using CX and Ry gates");
+  pool_m.def(
+      "CU1_using_CX", &CircPool::CU1_using_CX,
+      "Equivalent to CU1, using CX and U1 gates");
+  pool_m.def(
+      "CU3_using_CX", &CircPool::CU3_using_CX,
+      "Equivalent to CU1, using CX, U1 and U3 gates");
+  pool_m.def(
+      "ISWAP_using_CX", &CircPool::ISWAP_using_CX,
+      "Equivalent to ISWAP, using CX, U3 and Rz gates");
+  pool_m.def(
+      "XXPhase_using_CX", &CircPool::XXPhase_using_CX,
+      "Equivalent to XXPhase, using CX and U3 gates ");
+  pool_m.def(
+      "YYPhase_using_CX", &CircPool::YYPhase_using_CX,
+      "Equivalent to YYPhase, using CX, Rz and U3 gates");
+  pool_m.def(
+      "ZZPhase_using_CX", &CircPool::ZZPhase_using_CX,
+      "Equivalent to ZZPhase, using CX and Rz gates");
+  pool_m.def(
+      "XXPhase3_using_CX", &CircPool::XXPhase3_using_CX,
+      "Equivalent to 3-qubit MS interaction, using CX and U3 gates");
+  pool_m.def(
+      "ESWAP_using_CX", &CircPool::XXPhase3_using_CX,
+      "Equivalent to ESWAP, using CX, X, S, Ry and U1 gates");
+  pool_m.def(
+      "FSim_using_CX", &CircPool::FSim_using_CX,
+      "Equivalent to Fsim, using CX, X, S, U1 and U3 gates ");
+  pool_m.def(
+      "PhasedISWAP_using_CX", &CircPool::PhasedISWAP_using_CX,
+      "Equivalent to PhasedISWAP, using CX, U3 and Rz gates");
+  pool_m.def(
+      "NPhasedX_using_CX", &CircPool::NPhasedX_using_CX,
+      "Unwrap NPhasedX, into number_of_qubits PhasedX gates");
+
+  pool_m.def(
+      "tk1_to_PhasedXRz", &CircPool::tk1_to_PhasedXRz,
+      "A tk1 equivalent circuit given tk1 parameters in terms of PhasedX, Rz");
+  pool_m.def(
+      "tk1_to_rzrx", &CircPool::tk1_to_rzrx,
+      "A tk1 equivalent circuit given tk1 parameters in terms of Rz, Rx");
+  pool_m.def(
+      "tk1_to_rzh", &CircPool::tk1_to_rzh,
+      "A tk1 equivalent circuit given tk1 parameters in terms of Rz, H");
+  pool_m.def(
+      "tk1_to_rzsx", &CircPool::tk1_to_rzsx,
+      "A tk1 equivalent circuit given tk1 parameters in terms of Rz, Sx");
+  pool_m.def(
+      "tk1_to_tk1", &CircPool::tk1_to_tk1,
+      "A circuit of a single tk1 gate with given parameters");
```

Comment:
here you use `rx` and `rz` whereas previously these were capitalised: `Rz` and `Rx`. We should probably choose one or the other

Reply:
done